### ISSUE TITLE
Newly-tamed rat AI broken on multiplayer (1.16.4)

### ISSUE THREAD

**Issue by englens** [Alex-the-666/Rats#515]
Discovered and replicated on the pack Limitless 3. Shortly after taming, rats cease all movement and won't respond to any commands (follow, transport items, etc). They are still able to be picked up and placed down.

This probem persists until the server is rebooted, after which all problems clear up.

**Comment by BlueDuckYT** [comment]
You can use a rat sack to pick up and re-place them to fix this temporarily

**Comment by Crazydude323** [comment]
I tried the rat sack, but they still didn't work. For me it seems like the rats occasionally work, and then they just... stop working, for a long period of time. Hopefully they'll be fixed the next time I restart the server, but it definitely seems like something I'll have to wait for an update to fix.

**Comment by Sirplop** [comment]
Experiencing this bug in a custom modpack as well - it makes Rats pretty horrible and oftentimes the worst option to use. Picking them up with a rat sack fixes the problem, however changing their orders breaks their pathing again and they must be picked up again.

**Comment by Hermsmegerms** [comment]
issue also present in All the mods 6 on my server

**Comment by KieroVance** [comment]
I have this issue in single player, once tamed, they just... Don't do anything, I got one to move back and forth once... not on purpose, as I was trying to get it to milk a cow and it just moved a bunch until it got to a random block and stayed.

they don't move, work on their jobs, or anything, even on wander, in fact quitting and starting the game or the bag thing just...doesn't work for me...

basically, rats are just...broken...they're literally useless as once tamed, they just stay in one spot.

I have hte CREATE mod, and ratlantis, but thats it... and a chunk loader mod, but I haven't even used that yet...

I really hope this gets fixed, I made a whole area dedicated to this mod because I thought it'd be fun, but just... nothing...

**Comment by GizmoTheMoonPig** [comment]
Fixed in 5e5583927d7cc0727b0c8dbb162f274c0380c77e

**Comment by TheViralDivinity** [comment]
> Fixed in [5e55839](https://github.com/AlexModGuy/Rats/commit/5e5583927d7cc0727b0c8dbb162f274c0380c77e)

I'm sorry- but fixed my ass. To this day, ever since 2022, multiple times my rats just stop doing orders, and move to a random block. Transportation doesn't work as well. Period. Good idea, horrible execution for the mod if it's this buggy.
